recipe/meta.yaml

```diff
@@ -12,8 +12,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  skip: True  # [win]
+  number: 1
+  skip: True  # [win and py!=36]
   features:
     - vc14  # [win]
   
```

Comment:
You need `- python  # [win]` in build for the python/vc matrix

Reply:
Yes, just caught that. Thanks @isuruf!